### ISSUE TITLE
RightsStatementAttributeRenderer crashes when :rights_statement isn't valid

### ISSUE THREAD

**Issue by bess** [samvera/hyrax#2323]
### Descriptive summary

In Hyrax 2.0.0, I cannot display a work when `rights_statement == [""]`. It causes an uncaught exception.

### Expected behavior

I expect that if a work has an empty string in `rights_statement` it would either display an empty string or discard the value and not display anything.

### Actual behavior

The page will not render and the application reports an exception.

### Steps to reproduce the behavior

1. Make a work
1. Give it an empty string for it's rights statement: `work.rights_statement == [""]`
1. Try to display that work on a show view

We have fixed this locally by overriding the RightsStatementAttributeRenderer like this:

```
# This is used by PresentsAttributes to show licenses
#   e.g.: presenter.attribute_to_html(:rights_statement, render_as: :rights_statement)
class TuftsRightsStatementAttributeRenderer < Hyrax::Renderers::RightsStatementAttributeRenderer
  private

    ##
    # Special treatment for license/rights.  A URL from the Hyrax gem's config/hyrax.rb is stored in the descMetadata of the
    # curation_concern.  If that URL is valid in form, then it is used as a link.  If it is not valid, it is used as plain text.
    def attribute_value_to_html(value)
      return if value == ""
      begin
        parsed_uri = URI.parse(value)
      rescue
        nil
      end
      if parsed_uri.nil?
        ERB::Util.h(value)
      else
        %(<a href=#{ERB::Util.h(value)} target="_blank">#{Hyrax.config.rights_statement_service_class.new.label(value)}</a>)
      end
    rescue => exception
      Rails.logger.error("Could not render rights statement #{value}: #{exception}")
    end
end
```

**Comment by no-reply** [comment]
I think this is a duplicate of #2018, and stands to be closed by #2583.

**Comment by no-reply** [comment]
Closing this as fixed in #2583
